### ISSUE TITLE
Protected inherited members not documented in derived class if protected functions in base class is not documented (Origin: bugzilla #751453)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5813]
status NEW severity *normal* in component *documentation* for *---*
Reported in version *1.8.9.1* on platform *Other*
Assigned to: Dimitri van Heesch

Original attachment names and IDs:
- _[file_751453.txt](https://sourceforge.net/projects/doxygen/files/attachments/bug751453_306035_file_751453.txt/download/download)_ (ID 306035)

On 2015-06-24 18:17:06 +0000, scott.zhong@roguewave.com wrote:

    Created attachment 306035
    testcase header
    
    Protected inherited members not documented in derived class if protected
    functions in base class is not documented.
    
    configuration change:
    
    < HIDE_UNDOC_MEMBERS     = NO
    ---
    > HIDE_UNDOC_MEMBERS     = YES
    
    In Doxygen 1.8.3.1, Test class's documentation correctly documents inherited
    function Base::func().
    
    In Doxygen 1.8.9.1, Test class's documentation does not document the
    inherited function.

**Comment by abbeybreeding** [comment]
This is still and issue as of 1.11.10

**Comment by doxygen** [comment]
The original example was:
```c++
/**
 *  Base class
 */
class Base
{
public:
    /**
     * base func
     */
    void func();

protected:
    void pro();
};

/**
 * Test class
 */
class Test : protected Base
{
};
```
And the config file `doxygen -x`
```
# Difference with default Doxyfile 1.12.0 (657295e307c50e20c89a2bb00d45641868b4cc35)
HIDE_UNDOC_MEMBERS     = YES
```

The output shows an empty section for additional members (so when counting doxygen concludes there is something to show, but then it isn't actually rendered)
<img width="331" alt="image" src="https://github.com/doxygen/doxygen/assets/1300762/802c1035-132d-418a-a981-71865325dfce">

**Comment by doxygen** [comment]
@abbeybreeding Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official release becomes available.

**Comment by abbeybreeding** [comment]
@doxygen Hi Dmitri!

This does fix our issue. Thank you so much!  :)

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
